registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/SyncRegistrationService.java

```diff
@@ -53,6 +54,7 @@
 	
 	 public SyncRegistrationEntity findByRegistrationIdAndProcessAndIteration(String registrationId,String process,int iteration);
 
+	public SubWorkflowMappingEntity findWorkflowMappingByIdProcessIteration(String id, String process, int iteration);
```

Comment:
Can we follow the above away of naming method with And in between Id, process and iteration ?

Reply:
Also can we not add this method to RegistrationStatusServiceImpl, since it is rather fitting with registration table, rather than sync table ?